docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+import ExampleInputInverse from '../examples/exampleInputInverse';
+import ExampleInputLabel from '../examples/exampleInputLabel';
+import ExampleInputLabelPosition from '../examples/exampleInputLabelPosition';
+import ExampleInputLoading from '../examples/exampleInputLoading';
+import ExampleInputMaxMin from '../examples/exampleInputMaxMin';
+import ExampleInputMaxLength from '../examples/exampleInputMaxLength';
+import ExampleInputMinLength from '../examples/exampleInputMinLength';
+import ExampleInputOnBlur from '../examples/exampleInputOnBlur';
+import ExampleInputOnChange from '../examples/exampleInputOnChange';
+import ExampleInputOnClick from '../examples/exampleInputOnClick';
+import ExampleInputOnFocus from '../examples/exampleInputOnFocus';
+import ExampleInputOnKeyDown from '../examples/exampleInputOnKeyDown';
+import ExampleInputPlaceholder from '../examples/exampleInputPlaceholder';
+import ExampleInputRequired from '../examples/exampleInputRequired';
+import ExampleInputType from '../examples/exampleInputType';
+import ExampleInputMask from '../examples/exampleInputMask';
+import Heading from '../../../global/heading';
+import Main from '../../../global/main';
+import MarkdownContainer from '../../../global/markdownContainer';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as componentDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/input/input';
+/* eslint-enable import/no-named-default, import/extensions */
+
+function InputDevSandbox() {
+    const {
+        description: componentDescription,
+        displayName: componentName,
+    } = componentDoc;
+
+    return (
+        <Main page={camelCase(componentName)}>
+            <Main.Content>
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
+                    >
+                        {componentDescription}
+                    </Typography>
+
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Auto Complete
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Indicates whether the value of the control can be automatically completed by the browser.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputAutocomplete').default}
+                >
+                    <ExampleInputAutocomplete />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Auto Focus
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"auto-focus"`

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)

Reply:
Looks good.  Resolving.